sphinx/ext/intersphinx.py

```diff
@@ -340,21 +342,36 @@ def setup(app):
     return {'version': sphinx.__display_version__, 'parallel_read_safe': True}
 
 
-if __name__ == '__main__':
-    # debug functionality to print out an inventory
-    import sys
+def debug(argv):
+    """Debug functionality to print out an inventory"""
+    if len(argv) < 2:
+        msg = dedent("""
```

Comment:
That style will not preserve the newline. I think it reads better to have "Error:" at the start of the line. Would you like me to do this, instead?

```python 
print("Print out an inventory file.\n"
      "Error: ...",
      file=sys.stderr)
```